### PR TITLE
fix: add type fixes to fix 'prepare' scripts

### PR DESCRIPTION
**Pull request by david-mcafee**
*Issue #, if available:*

*Description of changes:*
Currently, the prepare script fails because of a type error. This PR resolves this issue.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/aws-appsync/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-appsync",
-  "version": "4.1.1",
+  "version": "4.1.2",
```
Comment (elorzafe) [inline packages/aws-appsync/package.json:3]: I think this is not necessary, lerna can do that automatically

File: packages/aws-appsync-subscription-link/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-appsync-subscription-link",
-  "version": "2.2.3",
+  "version": "2.2.4",
```
Comment (elorzafe) [inline packages/aws-appsync-subscription-link/package.json:3]: I think this is not necessary, lerna will do it automatically

**Review by elorzafe — CHANGES_REQUESTED** [review]
I would revert package version bump, lerna will do it when doing the release

File: packages/aws-appsync/package.json (hunk 1 above)
Comment (david-mcafee) [reply packages/aws-appsync/package.json:3]: Will remove!

File: packages/aws-appsync-subscription-link/package.json (hunk 2 above)
Comment (david-mcafee) [reply packages/aws-appsync-subscription-link/package.json:3]: Will remove!

**Review by elorzafe — APPROVED** [review]
LGTM! thanks @david-mcafee 🌮

**Pull request merged**